From: Muhammad Muzammil <m.muzzammilashraf@gmail.com>
Subject: mm: rmap.c: fix kernel-doc warning
Date: Fri, 6 Oct 2023 17:28:25 +0500

After running make htmldocs, getting this warning
warning: Function parameter or member 'folio' not described in
'folio_move_anon_rmap'
and this patch resolves this warning

Link: https://lkml.kernel.org/r/20231006122825.54840-1-m.muzzammilashraf@gmail.com
Signed-off-by: Muhammad Muzammil <m.muzzammilashraf@gmail.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Cc: Matthew Wilcox <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/rmap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/rmap.c~mm-rmapc-fix-kenrel-doc-warning
+++ a/mm/rmap.c
@@ -1129,7 +1129,7 @@ int folio_total_mapcount(struct folio *f
 
 /**
  * folio_move_anon_rmap - move a folio to our anon_vma
- * @page:	The folio to move to our anon_vma
+ * @folio:  The folio to move to our anon_vma
  * @vma:	The vma the folio belongs to
  *
  * When a folio belongs exclusively to one process after a COW event,
_
